src/promptflow/tests/sdk_cli_azure_test/e2etests/test_cli.py

```diff
@@ -0,0 +1,49 @@
+import os
+import os.path
+import sys
+from pathlib import Path
+
+import pytest
+
+from promptflow._cli._pf.entry import main
+
+FLOWS_DIR = "./tests/test_configs/flows"
+RUNS_DIR = "./tests/test_configs/runs"
+CONNECTIONS_DIR = "./tests/test_configs/connections"
+DATAS_DIR = "./tests/test_configs/datas"
+
+
+# TODO: move this to a shared utility module
+def run_pf_command(*args, cwd=None):
+    """Run a pf command with the given arguments and working directory.
+
+    There have been some unknown issues in using subprocess on CI, so we use this function instead, which will also
+    provide better debugging experience.
+    """
+    origin_argv, origin_cwd = sys.argv, os.path.abspath(os.curdir)
+    try:
+        sys.argv = ["pf"] + list(args)
+        if cwd:
+            os.chdir(cwd)
+        main()
+    finally:
+        sys.argv = origin_argv
+        os.chdir(origin_cwd)
+
+
+@pytest.mark.cli_test
+@pytest.mark.e2etest
+class TestCli:
+    # PF cli test is here because when designate connection provider to remote, we need azure dependencies.
+    def test_pf_flow_test(self, remote_workspace_resource_id):
```

Comment:
Could you add a test case about chat flow
You can refer to this test case https://github.com/microsoft/promptflow/blob/6260a318fe46ebb1ed84ca9127c6777e66844362/src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py#L766C1-L766C1